codectrl-server/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0"
-codectrl-protobuf-bindings = { git = "https://github.com/Authentura/codectrl-rust-protobuf-bindings", tag = "v0.8.0" }
+codectrl-protobuf-bindings = { git = "https://github.com/z9fr/codectrl-rust-protobuf-bindings", tag = "v0.8.0" }
```

Comment:
Can you make a separate PR for your changes to this too before we merge for this repository?